packages/e2e-test-utils/src/inserter.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Internal dependencies
+ */
+import { pressKeyWithModifier } from './press-key-with-modifier';
+
+/**
+ * Opens the global block inserter.
+ */
+export async function openGlobalBlockInserter() {
+	if ( ! ( await isGlobalInserterOpen() ) ) {
+		await toggleGlobalBlockInserter();
+
+		// Waiting here is necessary because sometimes the inserter takes more time to
+		// render than Puppeteer takes to complete the 'click' action
+		await page.waitForSelector( '.block-editor-inserter__menu' );
+	}
+
+	// Select the block tab by default if the tabs are visible
```

Comment:
>if the tabs are visible

When are they not?

Reply:
Nevermind, I see now:

https://github.com/WordPress/gutenberg/blob/70f65da753cac214b20b7b69549d41e0a1cb0db9/packages/block-editor/src/components/inserter/menu.js#L79